components/header/components/DotNavHeader.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const DotNavHeader = (): JSX.Element => {
+    return (
+        <ul className="list-none p-0">
+            <li className="py-2 px-2 mr-5 font-bold inline-block active-item">
+                <a href="#">Home</a>
+            </li>
+            <li className="py-2 px-2 mr-5 font-bold text-gray inline-block">
+                <a href="#">Realease & LTS</a>
+            </li>
```

Comment:
Create a component in the same file so you don't have to repeat so much classes.